Big-Spender/readme.md

```diff
@@ -48,7 +48,7 @@ You are working with Claire and Farnoosh, who are trying to complete a missing r
 **You:** Absolutely. Here's the SQL query you need:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT * FROM spends WHERE amount between 30000 AND 31000;
```

Comment:
It's not a good practice to select all columns available in a table. You might have issues in the future if someone creates a new column that holds lots of data that breaks your old query.

It's better to explicitly list the columns you need after the SELECT command.